### PR TITLE
Security headers for local development

### PR DESCRIPTION
**Pull request by mozbhearsum**
This is largely the same as what we'll use in stage/prod, except where noted to be different. I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1571095 for applying the appropriate parts to stage.

### CODE REVIEW

**Comment by mozbhearsum** [comment]
I just dug into unsafe-inline a bit more and it turns out my React Developer Tools addon was causing it to be required, not the app itself. I also discovered we need frame-src for token refreshes (although I don't think the lack of it was causing #16

File: .neutrinorc.js [hunk 1]
```diff
@@ -17,6 +17,29 @@ module.exports = {
...
+                       (which can't be locked down to a specific path due to CSP format)
+              style-src: https://fonts.googleapis.com for dynamically loaded fonts
+              font-src: https://fonts.gstatic.com for fonts!
+              connect-src: https://localhost:9000 for the backend api
+                           {ws,https}://localhost:8010 for webpack connections (local dev only)
+                           https://balrog-localdev.auth0.com for authentication.
+                           note: this is different in stage/prod
+              frame-src: https://balrog-localdev.auth0.com for background token refreshes
+            */
+            'Content-Security-Policy': "default-src 'none'; script-src 'self' 'unsafe-eval'; img-src 'self' https://*.gravatar.com https://*.githubusercontent.com https://i1.wp.com; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com; font-src https://fonts.gstatic.com; connect-src http://localhost:9000 ws://localhost:9000 https://localhost:8010 https://balrog-localdev.auth0.com; frame-src: https://balrog-localdev.auth0.com",
```
Comment (helfi92) [inline .neutrinorc.js]: For places where we have `localhost:9000`, I think we could just use `self` instead.

Comment (mozbhearsum) [reply .neutrinorc.js]: Good point. I'll do some double checking to make sure this works with ws://, too.

**Comment by mozbhearsum** [comment]
Whoops, sorry. It seems I didn't retest after adding frame-src (which I bungled). Should be fixed now.

**Comment by mozbhearsum** [comment]
> Ah, we would also probably want to allow the endpoint for release revisions coming in from googleapis.

Good point, I'll add that.

**Pull request merged**